aea/test_tools/test_cases.py

```diff
@@ -763,16 +764,6 @@ def change_directory(cls, path: Path) -> None:
         """
         os.chdir(Path(path))
 
-    @classmethod
-    def _terminate_subprocesses(cls) -> None:
-        """Terminate all launched subprocesses."""
-        for process in cls.subprocesses:
-            if not process.returncode == 0:
-                poll = process.poll()
-                if poll is None:
-                    process.terminate()
-                    process.wait(timeout=TERMINATION_TIMEOUT)
-
```

Comment:
this is only called in `teardown_class` and unnecessary, as we will now raise a `subprocess.TimeoutExpired` in case we cannot terminate a subprocess in our `BaseAEATestCase`

Reply:
Please keep this method and add a deprecation warning. We will remove this in the pre release for V2